src/lora.cpp

```diff
@@ -18,7 +18,7 @@ Result<Unit, Errc> init_lora()
         "AT+UART=TIMEOUT,2000",
     };
 
-    for (const auto command : commands) {
+    for (const auto &command : commands) {
```

Comment:
komendy to są pointery, kopiowanie jest szybsze niż refka, ale luz nie trzeba zmieniac, uwaga na przyszlosc